ansible/README.md

```diff
@@ -212,6 +212,10 @@ machine to run vagrant in [playbooks/vagrant.yml](playbooks/vagrant.yml) but
 it simply installs Vagrant from https://releases.hashicorp.com/vagrant/2.2.5/vagrant_2.2.5_x86_64.deb
 and also [virtualbox from their web site](https://www.virtualbox.org/wiki/Downloads)
 
+## Vagrant setup guide - Windows
+
+To test the ansible scripts, you'll need to install Vagrant and Virtualbox from [here](https://www.vagrantup.com/intro/getting-started/index.html)
+
 ## Executing under vagrant
```

Comment:
Does this title also need to be edited to state it's Linux / MacOS X?

Reply:
yep, will do